dumper-integration-tests/redshift-tests/src/test/java/com/google/integration/SvvColumnsTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.integration;
+
+import static com.google.edwmigration.dumper.base.TestBase.CSV_PARSER;
+import static com.google.edwmigration.dumper.base.TestBase.assertMultisetsEqual;
+import static com.google.edwmigration.dumper.base.TestConstants.EXPORTED_FILES_BASE_PATH;
+import static com.google.edwmigration.dumper.base.TestConstants.PASSWORD_DB;
+import static com.google.edwmigration.dumper.base.TestConstants.SQL_REQUESTS_BASE_PATH;
+import static com.google.edwmigration.dumper.base.TestConstants.URL_DB;
+import static com.google.edwmigration.dumper.base.TestConstants.USERNAME_DB;
+
+import com.google.common.collect.LinkedHashMultiset;
+import com.google.edwmigration.dumper.pojo.SvvColumnsRow;
+import com.google.edwmigration.dumper.sql.SqlUtil;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import com.opencsv.exceptions.CsvValidationException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class SvvColumnsTest {
+
+  private static final String SQL_PATH = SQL_REQUESTS_BASE_PATH + "svv_columns.sql";
+  private static final String CSV_FILE_PATH = EXPORTED_FILES_BASE_PATH + "svv_columns.csv";
+  private static Connection connection;
+
+  @BeforeClass
+  public static void beforeClass() throws SQLException {
+    connection = DriverManager.getConnection(URL_DB, USERNAME_DB, PASSWORD_DB);
```

Comment:
This needs to be closed, because in a long-running test JVM, this static variable (and hence the connection) has a long lifetime. For a single-test class, I'd rather see it done in the t-w-r block below.

Reply:
What if I change it `@BeforeTest` that scope shouldn't have a long lifetime?

Reply:
No, because it's still static. The lifetime is until the class unloads. Changing it to @BeforeTest would probably not change the actual lifetime in a single-test class. Making it non-static will possibly reduce the lifetime, but not be correct.

Could we please have it in the t-w-r block unless there's a reason not to? Minimize extent.

Reply:
Done.